engine/test/cases/urlfield.test.js

```diff
@@ -0,0 +1,774 @@
+import * as Code from '@hapi/code'
+import * as Lab from '@hapi/lab'
+import UrlField from '../../components/urlfield'
+const lab = Lab.script()
+exports.lab = lab
+const { expect } = Code
+const { suite, test } = lab
+const joi = require('joi')
+
+// https://raw.githubusercontent.com/citizenlab/test-lists/master/lists/ae.csv
+const urls = [
```

Comment:
Maybe break this list out to a separate file? Just to aid developer readability of the actual test code going forward

Reply:
moved the list of test urls into a seperate js file.